mailexporter.go

```diff
@@ -470,15 +482,8 @@ func main() {
 	}
 
 	go detectAndMuxMail(fswatcher)
-
-	// now fire up the monitoring jobs
-	for _, c := range globalconf.Servers {
-		go monitor(c)
-
-		// keep a timedelta between monitoring jobs to reduce interference
-		// (although that shouldn't be an issue)
-		time.Sleep(globalconf.StartupOffset)
-	}
+	
+	startMonitoring()
```

Comment:
With this read, the start of the monitoring Jobs is moved into a separate function, but it still blocks until all monitoring-coroutines are started.

With a `go` in front this function call becomes a goroutine and is therefore no longer blocking further processing of `main()`. (Compare the function call two lines above.)